packages/google_sign_in/google_sign_in/README.md

```diff
@@ -61,6 +61,20 @@ This plugin requires iOS 9.0 or higher.
 <!-- End of the Google Sign-in Section -->
 ```
 
+### iOS Authenticate with backend server
+
+If you use Google Sign-In with an app that communicates with a backend server,
```

Comment:
Please add a link to the original doc for the usage of `SERVER_CLIENT_ID`. Maybe this: https://developers.google.com/identity/sign-in/ios/backend-auth#objective-c